opentelemetry-exporter-gcp-logging/README.rst

```diff
@@ -0,0 +1,82 @@
+OpenTelemetry Google Cloud Logging Exporter
+==============================================
+
+.. image:: https://badge.fury.io/py/opentelemetry-exporter-gcp-logging.svg
+    :target: https://badge.fury.io/py/opentelemetry-exporter-gcp-logging
+
+.. image:: https://readthedocs.org/projects/google-cloud-opentelemetry/badge/?version=latest
+    :target: https://google-cloud-opentelemetry.readthedocs.io/en/latest/?badge=latest
+    :alt: Documentation Status
+
+This library provides support for exporting logs to Google Cloud
+Logging.
+
+To get started with instrumentation in Google Cloud, see `Generate traces and metrics with
+Python <https://cloud.google.com/stackdriver/docs/instrumentation/setup/python>`_.
+
+To learn more about instrumentation and observability, including opinionated recommendations
+for Google Cloud Observability, visit `Instrumentation and observability
+<https://cloud.google.com/stackdriver/docs/instrumentation/overview>`_.
+
+For resource detection and GCP trace context propagation, see
+`opentelemetry-tools-google-cloud
+<https://pypi.org/project/opentelemetry-tools-google-cloud/>`_. For the
+Google Cloud Trace exporter, see `opentelemetry-exporter-gcp-trace
+<https://pypi.org/project/opentelemetry-exporter-gcp-trace/>`_.
+
+Installation
+------------
+
+.. code:: bash
+
+    pip install opentelemetry-exporter-gcp-logging
+
+Usage
+-----
+
+.. code:: python
+
+    from opentelemetry.exporter.cloud_logging import (
```

Comment:
Could you update this sample to set it up with the logging SDK instead of calling directly? Here's an example with OTLP exporter you could start with https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/examples/logs/example.py

Reply:
Mostly looks good. But the Logger you get is just a standard python logger class, you can just use it as usual. You don't need to send OTel `LogRecords` into it https://docs.python.org/3/howto/logging.html#a-simple-example. 

The purpose of the OTel `LoggingHandler` is to capture the stdlib `logging` logs into OTel data model.

If you want to add some structured logs, you use `extra` kwarg to any logger method https://docs.python.org/3/library/logging.html#:~:text=The%20fourth%20keyword,messages.%20For%20example%3A